example/data-migration-ecs/variables.tf

```diff
@@ -0,0 +1,38 @@
+variable "prefix" {
+  type = string
+  default = null
+}
+
+variable "region" {
+  type    = string
+  default = "us-east-1"
+}
+
+variable "ecs_cluster_arn" {
+  type    = string
+  default = null
+}
+
+variable "permissions_boundary_arn" {
+  type    = string
+  default = null
+}
+
+variable "subnet_ids" {
+  type = list(string)
+}
+
+variable "vpc_id" {
+  type    = string
+  default = null
+}
+
+variable "log2elasticsearch_lambda_function_arn" {
+  type    = string
+  default = null
+}
+
+variable "data_migration2_function_arn" {
+  type    = string
+  default = null
+}
```

Comment:
newline

Reply:
Yeah, it looks like my linter isn't evaluating .tf files. Maybe we should fix that so this throws an error/warning?